cerbero/utils/__init__.py

```diff
@@ -265,6 +265,10 @@ def system_info():
                 distro_version = DistroVersion.DEBIAN_STRETCH
             elif d[1].startswith('10.') or d[1].startswith('buster'):
                 distro_version = DistroVersion.DEBIAN_BUSTER
+            elif d[0] in ['debian', 'Debian GNU/Linux']:
+                distro_version = "debian_{number:02d}_{name}".format(number=int(d[1]), name=d[2])
+            elif d[0] in ['Ubuntu']:
+                distro_version = "ubuntu_{number}_{name}".format(number=d[1].replace('.', '_'), name=distro_version)                
```

Comment:
I meant here
> Why not use the same logic with enums like with other distros?

Reply:
> We no longer add enums manually for every distro because it requires updating every 6 months, and older cerbero versions don't work when new distro releases happen for no particular reason.

from community PR https://gitlab.freedesktop.org/gstreamer/cerbero/-/merge_requests/845#note_1339467